flink-connectors/flink-connector-kafka-base/src/test/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumerBaseTest.java

```diff
@@ -250,6 +251,118 @@ public void testConfigureDisableOffsetCommitWithoutCheckpointing() throws Except
 		assertEquals(OffsetCommitMode.DISABLED, consumer.getOffsetCommitMode());
 	}
 
+	/**
+	 * Tests that subscribed partitions are the same when restored partitions
+	 * and discovered partitions didn't change.
+	 * (filterRestoredPartitionsWithDiscovered is active)
+	 */
+	@Test
+	public void testSetFilterRestoredParitionsNoChange() throws Exception {
+		checkFilterRestoredPartitionsWithDisovered(Arrays.asList(new String[]{"kafka_topic_1", "kafka_topic_2"}),
```

Comment:
nit: I would also let the first argument be a new line, just for consistency and readability.
Same for the other tests.

Reply:
Done.